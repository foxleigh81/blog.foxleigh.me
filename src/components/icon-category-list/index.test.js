import React from 'react'
import renderer from 'react-test-renderer'

import ICL from './index'

const icons = {
  allIcons: {
    edges: [
      {
        node: {
          id: 'a2599ceb-0465-5632-ac77-76fe8e0a538d',
          category: 'additional',
          name: 'add-circle-outline',
          path: 'additional/add-circle-outline.svg',
        },
      },
      {
        node: {
          id: '91ad58fd-af30-5533-a358-19c08977e063',
          category: 'additional',
          name: 'add-circle',
          path: 'additional/add-circle.svg',
        },
      },
    ],
  }
}

describe('Icon Category List', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ICL icons={icons} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
