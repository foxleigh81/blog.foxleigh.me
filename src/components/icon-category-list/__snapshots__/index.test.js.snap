// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Category List renders correctly 1`] = `
<section
  className="icon-list"
>
  <h1>
    additional
  </h1>
  <ul>
    <li>
      <img
        alt="add-circle-outline"
        src="test-file-stub"
      />
    </li>
    <li>
      <img
        alt="add-circle"
        src="test-file-stub"
      />
    </li>
  </ul>
</section>
`;
